app/assets/stylesheets/custom.css.scss

```diff
@@ -27,7 +27,7 @@ body {
 }
 
 .navbar {
-  background-color: #D23200
+  background-color: #D23200;
```

Comment:
Color literals like `#D23200` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D23200` should be written as `#d23200`